package/crossplane.yaml.tmpl

```diff
@@ -1,24 +1,28 @@
 apiVersion: meta.pkg.crossplane.io/v1alpha1
 kind: Provider
 metadata:
-  name: provider-gcp-{{ APIGROUP }}
+  name: {{ .Name }}
+{{ if ne .Service "monolith" }}
   labels:
-    pkg.crossplane.io/provider-family: gcp
+    pkg.crossplane.io/provider-family: provider-family-gcp
+{{ end }}
   annotations:
     meta.crossplane.io/maintainer: Upbound <support@upbound.io>
     meta.crossplane.io/source: github.com/upbound/provider-gcp
     meta.crossplane.io/description: |
       Upbound's official Crossplane provider to manage Google Cloud Platform (GCP)
-      {{ APIGROUP }} services in Kubernetes.
+      {{ .Service }} services in Kubernetes.
     meta.crossplane.io/readme: |
-      Provider GCP is a Crossplane provider for [Google Cloud Platform
-      (GCP)](https://cloud.google.com) developed and supported by Upbound.
+      Provider GCP is a Crossplane provider for [Google Cloud Platform](https://cloud.google.com/) developed and supported by Upbound.
       Available resources and their fields can be found in the [Upbound
       Marketplace](https://marketplace.upbound.io/providers/upbound/provider-gcp).
       If you encounter an issue please reach out on support@upbound.io email
-      address. This is a subpackage for the {{ APIGROUP }} API group.
-    friendly-name.meta.crossplane.io: Provider GCP ({{ APIGROUP }})
+      address. This is a subpackage for the {{ .Service }} API group.
+    friendly-name.meta.crossplane.io: Provider GCP ({{ .Service }})
+    auth.upbound.io/group: gcp.upbound.io
+{{ if and (ne .Service "config") (ne .Service "monolith") -}}
 spec:
   dependsOn:
-    - provider: {{ XPKG_REG_ORGS }}/provider-gcp-config
-      version: "{{ DEP_CONSTRAINT }}"
+    - provider: {{ .XpkgRegOrg }}/provider-family-gcp
```

Comment:
If this what we finally aligned in terms of naming the config package?

Reply:
Done. I renamed the family package from`provider-family-gcp` to `provider-gcp-config`, which is consistent with our smaller provider naming scheme.